shared/util/quit-helper.desktop.js

```diff
@@ -47,7 +56,8 @@ function _executeActionsFromRenderer (actions: Array<Action>) {
   ipcRenderer.send('executeActions', actions)
 }
 
-export function executeActions (actions: Array<Action>) {
+export function executeActionsForContext (context: Context) {
```

Comment:
I'm just saying the idea that there are 'parts' isn't required for this
logic now. I know if we have types and payloads and etc we'll want a
structure like this but we don't have a need for that currently

On Friday, July 15, 2016, Marco Munizaga <notifications@github.com> wrote:

> In shared/util/quit-helper.desktop.js
> <https://github.com/keybase/client/pull/3469#discussion_r71048888>:
>
> > @@ -47,7 +56,8 @@ function _executeActionsFromRenderer (actions: Array<Action>) {
> >    ipcRenderer.send('executeActions', actions)
> >  }
> >
> > -export function executeActions (actions: Array<Action>) {
> > +export function executeActionsForContext (context: Context) {
>
> I disagree with this not making the code simpler because it would be
> decoupling parts.
>
> but it's also not a big change, so it's fine for now
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/keybase/client/pull/3469/files/696a85227ca0f7720ffeeea0c9eb4a149973b035#r71048888>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAfhU_H9fRkrKCtd2rWL9OTRXRPK5ZK0ks5qWAh5gaJpZM4JMsjQ>
> .
>